scripts/deploy-production.js

```diff
@@ -0,0 +1,126 @@
+const prompts = require("prompts");
+const { exec } = require("child_process");
+
+const localDeployScript =
+  "yarn build:production && env-cmd -e production yarn deploy:do-not-use && env-cmd -e production ./scripts/email.sh";
```

Comment:
seems like this should say `yarn build:prod`

Also wanted to mention that the `cmdrc` field in Spruce is called `prod`, but in Parsley it is called `production`. If you want to keep it as is that's fine with me though, as you would have to change it in multiple places